docs/data/material/customization/theme-components/theme-components.md

```diff
@@ -77,17 +90,62 @@ const finalTheme = createTheme({
 
 {{"demo": "GlobalThemeOverrideCallback.js"}}
 
-### Using `sx` (experimental) syntax
+### Using the `sx` experimental syntax
+
+If you are not familiar with the `sx` prop, make sure you first check out [the concept around it](/system/the-sx-prop/) and [the difference with the `styled` utility](/system/styled/#difference-with-the-sx-prop).
```

Comment:
Trimming down & grammar.

```suggestion
If you are not familiar with the `sx` prop, learn more about [the concept behind it](/system/the-sx-prop/) and [how it differs from the `styled` utility](/system/styled/#difference-with-the-sx-prop).
```

Reply:
On second thought, I think we should replace this sentence with a very brief explanation of the `sx` prop so the reader doesn't have to leave this page.

Reply:
Yup, I like that! I easily get exhausted when reading a piece of text that just several reference links... end up feeling like I have to go through all of them first before finishing the initial one I started with 😅 

How about that?

```suggestion
If you are not familiar with the `sx` prop, it consists of a shortcut for defining custom styles that access the theme object. The property is a superset of the CSS that packages all the style functions. To learn more about [the concept behind it](/system/the-sx-prop/) and [how it differs from the `styled` utility](/system/styled/#difference-with-the-sx-prop).
```